package.json

```diff
@@ -24,7 +24,7 @@
     "Nate Cohen-Smith <ncohensm@cisco.com>",
     "Vagou Zhou <vagouz@cisco.com>"
   ],
-  "main": "src/index.js",
+  "main": "packages/web-client-internal/dist/index.js",
```

Comment:
top level shouldn't have a `main`. to link into the web client, run `npm link` from the web-client-internal directory.